### PR TITLE
Removed some unnecessary calculations

### PR DESCRIPTION
**Pull request by virophagesp**
some additions and divisions were not necessary as they were just 2 constant numbers
at first i was checking addition but then i realised it would be a better idea to look at division

### CODE REVIEW

**Comment by fire** [comment]
Focusing on code readability and context ensures better understanding and maintainability, especially when performance improvements are not measured. 

This pr attempts to optimize but there are no metrics.

**Comment by virophagesp** [comment]
I do not see how it has become less readable?

File: editor/editor_themes.cpp [hunk 1]
```diff
@@ -581,10 +581,10 @@ Ref<Theme> create_editor_theme(const Ref<Theme> p_theme) {
 	const float prop_color_saturation = accent_color.get_s() * 0.75;
 	const float prop_color_value = accent_color.get_v();
 
-	theme->set_color("property_color_x", "Editor", Color().from_hsv(0.0 / 3.0 + 0.05, prop_color_saturation, prop_color_value));
+	theme->set_color("property_color_x", "Editor", Color().from_hsv(0.05, prop_color_saturation, prop_color_value));
 	theme->set_color("property_color_y", "Editor", Color().from_hsv(1.0 / 3.0 + 0.05, prop_color_saturation, prop_color_value));
 	theme->set_color("property_color_z", "Editor", Color().from_hsv(2.0 / 3.0 + 0.05, prop_color_saturation, prop_color_value));
-	theme->set_color("property_color_w", "Editor", Color().from_hsv(1.5 / 3.0 + 0.05, prop_color_saturation, prop_color_value));
+	theme->set_color("property_color_w", "Editor", Color().from_hsv(0.55, prop_color_saturation, prop_color_value));
```
Comment (aaronfranke) [inline editor/editor_themes.cpp:587]: This is worse. The old code has clearer meaning (halfway between one-third and two-thirds hue plus 0.05).

The compiler will automatically perform these calculations at compile time, there is no performance impact to leaving the expressions as they were before.

File: thirdparty/libwebp/src/dec/frame_dec.c [hunk 2]
```diff
@@ -694,7 +694,7 @@ static int AllocateMemory(VP8Decoder* const dec) {
   const size_t mb_data_size =
       (dec->mt_method_ == 2 ? 2 : 1) * mb_w * sizeof(*dec->mb_data_);
   const size_t cache_height = (16 * num_caches
-                            + kFilterExtraRows[dec->filter_type_]) * 3 / 2;
+                            + kFilterExtraRows[dec->filter_type_]) * 1.5;
```
Comment (aaronfranke) [inline thirdparty/libwebp/src/dec/frame_dec.c:697]: 1. We do not ever change things in third-party code.
2. You changed behavior here, this used to be integer division.

**Comment by akien-mga** [comment]
@virophagesp This is not the first time you make such unprompted refactoring PRs which end up rejected as they don't actually improve things.

If you want to contribute to improve the codebase, I suggest you join the [contributors chat](chat.godotengine.org) to discuss your ideas and whether they're worth pursuing beforehand.

In general PRs should fix a reported bug, or implement a user requested proposal. Drive-by code style or non benchmarked microoptimization changes by first time contributors rarely end up merged.

**Pull request closed without merging**